app/services/foods_service.rb

```diff
@@ -0,0 +1,39 @@
+class FoodsService
+
+  def self.search_ingredients(ingredient)
+    paramiters = {query: ingredient}
+    response = connection.get('fdc/v1/foods/search') do |faraday|
+      paramiters.each do |k, v|
+        faraday.params[k] = v
+      end
+    end
+
+    parse_json(response)
+  end
+
+
+  def self.food_list(ingredient, list_length = 10) # move to facade in future
+    food_data = search_ingredients(ingredient)
+    top_ten = food_data[:foods].first(list_length)
+    # binding.pry
+    future_poro = top_ten.map do |food|#in future make this poro
+      {brand: food[:brandOwner],
+      description: food[:lowercaseDescription],
+      gtinUpc: food[:gtinUpc],
+      ingredients: food[:ingredients]}
+    end
+    future_poro
+
+  end
```

Comment:
Thanks for putting these notes in here -- helps me to see how you would ideally refactor it had you had more time.

Reply:
You could also grab the totalHits from `food_data`, and put that into the object you're returning to your controller.. that would help eliminate the amount of instance variables you're currently sending to the view.